.eslintrc

```diff
@@ -2,7 +2,7 @@
 parser: babel-eslint
 
 extends:
-  - ./node_modules/fbjs-scripts/eslint/.eslintrc
+  - ./node_modules/fbjs-scripts/eslint/.eslintrc.js
```

Comment:
This is going to conflict with https://github.com/facebook/relay/pull/1161

Can you please revert this bit or rebase and then we can import this.